src/components/Modal.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useRef, useEffect } from 'react';
+import ReactDom from 'react-dom';
+import Button from './Button';
+import styles from './Modal.module.scss';
+const modalRoot = document.getElementById('portal');
+
+const ModalDefault = ({ children, modalIsOpen, setModalIsOpen, ...props }) => {
```

Comment:
Props that are expected to hold a boolean value (`true`/`false`) should be prefixed with `is...`, in this case: `isModalOpen`. That way you can recognize a potential boolean value at the first glance.